obelisk_ws/src/robots/achilles_model/LICENSE

```diff
@@ -0,0 +1,17 @@
+Permission is hereby granted, free of charge, to any person obtaining a copy
```

Comment:
in a future PR, can we go and remove all the internal license files (or add licenses to all packages and make sure they're all the same as the outer one)